site/docs/v1/tech/admin-guide/upgrade-pre-1-37.adoc

```diff
@@ -0,0 +1,296 @@
+---
```

Comment:
Is this just a straight copy? i.e. I don't need to review it- this isn't new content?